docs/run-instance/running-with-docker.md

```diff
@@ -1,247 +1,509 @@
 
-1. Recommended: Run latest release version image using `docker compose`
+```shell
+wget https://github.com/GIScience/openrouteservice/releases/download/v8.0.0/docker-compose.yml
+```
 
-    ```shell
-    # For latest release build
-    wget https://raw.githubusercontent.com/GIScience/openrouteservice/main/docker-compose.yml
-    docker compose up -d
-    ```
-   This will pull the latest release build of openrouteservice from Docker Hub and start it up using an example setup
-   and the provided test OSM file for Heidelberg/Germany and surrounding area.
+Now start openrouteservice in the background:
+```shell
+docker compose up -d
+```
 
-2. Run specific version image using `docker compose`
+This will pull the openrouteservice of the selected version and start it up using an example setup
+and the provided test OSM file for Heidelberg/Germany and surrounding area.
 
-   You can specify the Docker image tag by declaring an environment variable named `ORS_TAG`. Possible values are
-   specific version tags such as `v8.0.0` or `nightly` for the newest nightly build.
+To see the container's logs, run
+```shell
+docker compose logs 
+```
+or 
+```shell
+docker compose logs -tf  
+```
+to follow the log.
 
-    ```shell
-    # For nightly builds
-    wget https://raw.githubusercontent.com/GIScience/openrouteservice/main/docker-compose.yml
-    export ORS_TAG=nightly && docker compose up -d
-    ```
+To stop the container again, execute
+```shell
+docker compose stop 
+```
+or
+```shell
+docker compose down
+```
+to stop and remove the container.
 
-## Build image
+The file `docker-compose.yml` contains plenty of commented options with descriptions. 
+You can un-comment those options and set your custom option values.
+
+::: details Running without docker compose
+It is easier to modify the Docker compose file, but to test a specific setup, you can also run a command similar to what docker compose would run directly on the command line. 
+Here is a basic example:
+```shell
+# create directories for volumes to mount as local user
+mkdir -p ors-docker/config ors-docker/elevation_cache ors-docker/graphs ors-docker/files ors-docker/logs 
+docker run -dt -u "${UID}:${GID}" \
+  --name ors-app \
+  -p 8080:8082 \
+  -v $PWD/ors-docker/config:/home/ors/config \
+  -v $PWD/ors-docker/elevation_cache:/home/ors/elevation_cache \
+  -v $PWD/ors-docker/graphs:/home/ors/graphs \
+  -v $PWD/ors-docker/files:/home/ors/files \
+  -v $PWD/ors-docker/logs:/home/ors/logs \
+  #-e "BUILD_GRAPHS=True" \
+  -e "JAVA_OPTS=-Djava.awt.headless=true -server -XX:TargetSurvivorRatio=75 -XX:SurvivorRatio=64 -XX:MaxTenuringThreshold=3 -XX:+UseG1GC -XX:+ScavengeBeforeFullGC -XX:ParallelGCThreads=4 -Xms1g -Xmx2g" \
+  -e "CATALINA_OPTS=-Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.port=9001 -Dcom.sun.management.jmxremote.rmi.port=9001 -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false -Djava.rmi.server.hostname=localhost" \
```

Comment:
```suggestion
  -e "XMS=1g" \
  -e "XMX=2g \
```

Reply:
Let's keep the JAVA_OPTS and CATALINA_OPTS out. Only xmx and xms are really important in this quick example.